.circleci/config.yml

```diff
@@ -1,16 +1,25 @@
-# Elixir CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-elixir/ for more details
+install_elixir: &install_elixir
+  run:
+    name: Install Elixir
+    command: |
+      wget https://repo.hex.pm/builds/elixir/v$ELIXIR_VERSION.zip
+      unzip -d /usr/local/elixir v$ELIXIR_VERSION.zip
+      echo 'export PATH=/usr/local/elixir/bin:$PATH' >> $BASH_ENV
+
 version: 2
 jobs:
   build:
     docker:
-      - image: circleci/elixir:1.7.3
```

Comment:
Since this is backwards compatible, I think we should keep running CI on 1.7.3 (or 1.7.4). The mix.exs also says that 1.6 works. Do we want to keep supporting that?

Reply:
I've updated the tests suite to run for the same version set as `nerves_runtime`. This ends up being a mix of otp 20 and 21 for ~> 1.6, 1.7, 1.8